### PR TITLE
absolute URL for media

### PR DESCRIPTION
**Pull request by itkg-rboessel**

### CODE REVIEW

**Comment by ngilain** [comment]
What about the rendering of media in the BO media gallery ? You should check modifications for that too

**Comment by itkg-rboessel** [comment]
@itkg-ngilain I install a local media vhost. That's works fine.

**Comment by itkg-rboessel** [comment]
https://github.com/open-orchestra/open-orchestra-media-bundle/pull/75

**Comment by ngilain** [comment]
@itkg-rboessel : I was not clear when I talked about BO gallery. I wanted to talk about the media preview gallery, and the thumbnails edit form.

Is that ok ?

File: MediaAdminBundle/Transformer/MediaTransformer.php [hunk 1]
```diff
@@ -74,4 +76,17 @@ public function getName()
     {
         return 'media';
     }
+
+    /**
+     * @param $key
+     *
+     * @return string
+     */
+    protected function generateAbsoluteRoute($key)
```
Comment (ngilain) [inline MediaAdminBundle/Transformer/MediaTransformer.php]: you could rename now into something like generateMediaUrl

**Comment by ngilain** [comment]
:+1:

**Pull request merged**